### ISSUE TITLE
Severely decreased tear production on Schirmer I test

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11645]
**Preferred term label:**
Severe decreased lacrimation on Schirmer I test

**Synonyms:**
* Schirmer I test less than 5 mm in 5 minutes
* Severe aqueous tear deficiency on Schirmer I test
* Wetting of less than 5 mm on Schirmer I test

**Definition (free text, please give PubMed ID):**
An abnormally low rate of tear production characterized by a Schirmer I test result of less than 5 millimeters of wetting on the standard filter paper strip over a 5-minute period, which is a clinical threshold widely used to define severe dry eye or severe aqueous tear deficiency. PMID: 32298795

**Parent term (use hpo.jax.org/app):**
Decreased lacrimation (HP:0000633)

**Your nano-attribution (ORCID):**
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 For this term I am wondering if the definition is in essence a definition of Decreased lacrimation, or is there is a standard clinical classification of Mild/Moderate/Severe that is in common enough use that all ophthalmologists are using it? Otherwise it is probably best to use the HPO Modifier terms to denote severity.
